README.md

```diff
@@ -64,11 +62,24 @@ heroku login # need credentials from Josh C.
 heroku pg:pull DATABASE_URL curriculumbuilder -a curriculumbuilder
 ```
 
-8. run the server
+8. set up local_settings.py
+
+```
+create curruculumbuilder/curriculumBuilder/local_settings.py
+copy starter settings from local_settings.py.example
```

Comment:
we should already be in the curriculumbuilder directory. so better to just list the exact command:
```
cp curriculumBuilder/local_settings.py.example curriculumBuilder/local_settings.py
```